src/neuroconv/tools/roiextractors/imagingextractordatachunkiterator.py

```diff
@@ -82,6 +82,18 @@ def __init__(
             progress_bar_options=progress_bar_options,
         )
 
+    def _get_default_chunk_shape(self, chunk_mb: float) -> tuple:
+        """Select the chunk_shape less than the threshold of chunk_mb while keeping the original image size."""
```

Comment:
@CodyCBakerPhD do you think this is clear enough? I want to make sure it's clear that we are keeping the image shape and fitting how many frames fit into `chunk_mb`?

Reply:
Yes, I think it makes perfect sense

Reply:
Maybe one day we will have to consider how to deal with super higher resolution images... but for now it always makes sense (both optimizing on disk and in cloud) to chunk by entire frame